README.md

```diff
@@ -121,12 +120,35 @@ following JSON specification:
 <a id="markdown-quick-start" name="quick-start"></a>
 ## Quick Start
 
-<Hello world style example.>
+### Pre-requisite
+
+* Install docker and docker-compose.
+* Read up on [serverfull](https://github.com/asecurityteam/serverfull)
+* Read up on [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
+
+The app can be run locally by running `make run`.
+
+This will run `docker-compose` for the serverfull project
+as well as the supplied serverfull-gateway configuration.
+The sample configration provided assumes there will be a stats
+collector running. To disable this, remove the stats configuration
+lines from the server configuration and the serverfull-gateway
+configuration.
+
+The app should now be running on port 8080.
+
+`curl -vX POST "http://localhost:8080" -H "Content-Type:application/json" -d @pkg/handlers/v1/testdata/ec2.0.json`
 
 <a id="markdown-configuration" name="configuration"></a>
 ## Configuration
 
-<Details of how to actually work with the project>
+Images of this project are built, and hosted on [DockerHub](https://cloud.docker.com/u/asecurityteam/repository/docker/asecurityteam/awsconfig-transformerd).
+
+This code functions as a stand-alone Lambda function, and can be deployed to AWS Lambda directly.
```

Comment:
This is true. However, we don't yet provide what a customer would need to do this. For example, we could offer an alternative `main.go` (or a flag in the _main_ main.go) that actually runs the lambda server from the AWS SDK.
 
The only thing missing from our setup is replacing our launching of the HTTP server with:
```golang
lambda.Start(myHandler.Handle)
```

Reply:
Very true. Given that we don't have great support for running in the AWS Lambda (e.g. we don't have lambda middleware which inject stats/logs), I can document something about forking the project and running the handler as you've stated above.
Once we get some more experience running actual AWS Lambdas, we can provide something with better long term support.

Reply:
I might, instead, recommend that folks import rather than fork if they are going to run with it before we offer support. That way it helps encourage them to make changes/improvements here rather than their fork and stages them to take advantage of our release that includes native lambda helpers.